data/virt_autotest/kubevirt_tests/full-tests.conf

```diff
@@ -92,17 +90,15 @@ GINKGO_FOCUS=services.go
 GINKGO_FOCUS=vmi_kernel_boot_test.go
 
 [infra_test]
-GINKGO_FOCUS=infra_test.go
-GINKGO_SKIP=test_id:4099|test_id:4137|test_id:4139|test_id:6535
+GINKGO_FOCUS=infrastructure.go
 
 [operator_test]
-GINKGO_FOCUS=operator_test.go
-GINKGO_SKIP=test_id:3153
+GINKGO_FOCUS=operator.go
 EXTRA_OPT=-skip-shasums-check
 
 [migration_test]
-GINKGO_FOCUS=migration_test.go
-GINKGO_SKIP=test_id:6979|test_id:6982|test_id:3190|test_id:4746
+GINKGO_FOCUS=migration.go
+GINKGO_SKIP=test_id:4746
```

Comment:
I do not quite understand -- this ticket involves only core-tests.conf, right? Why would full-tests.conf be modified ?

Reply:
> I do not quite understand -- this ticket involves only core-tests.conf, right? Why would full-tests.conf be modified ?

I also did testing on SLE, and some test cases are available for running in the latest version. So removing them from skipped list.